<style>
<!--
 li.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	font-size:10.5pt;
	font-family:"Times New Roman";
	margin-left:0cm; margin-right:0cm; margin-top:0cm}
-->
</style>

<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
</head>

<div>
	<table border="0" id="table1" bgcolor="#FFFFFF">
<tr>
	<td valign="baseline" colspan="3" bgcolor="#ADD8E6">
		<p><b>
			<font face="Times" size="3">
				<a name="2023">Protein Function Prediction</a>
			</font>
		</b></p>
	</td>
</tr>

		<tr>

		<td valign="baseline" colspan="3">
			<br>
			<font color="blue">Overview</font>
		<ol>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			深度学习在蛋白质功能预测中的应用.
			<b> 合成生物学</b>, 2023.
			[<a href="./papers/1/Chinese Overview.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Deep learning methods for protein function prediction.
			<b> Proteomics</b>, 2025.
			[<a href="./papers/1/Review_2025.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			A comprehensive computational benchmark for evaluating deep learning-based protein function prediction approaches.
			<b> Briefings in Bioinformatics</b>, 2024.
			[<a href="./papers/1/Overview_2024.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			A large-scale assessment of sequence database search tools for homology-based protein function prediction.
			<b> Briefings in Bioinformatics</b>, 2024.
			[<a href="./papers/1/English Overview2.pdf">PDF</a>]
			[<a href="https://github.com/kad-ecoli/blastKNN_benchmark/" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Deep learning methods for protein function prediction.
			<b> Proteomics</b>, 2024.
			[<a href="./papers/1/English Overview1.pdf">PDF</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein function prediction with gene ontology: from traditional to deep learning models.
			<b> PeerJ</b>, 2021.
			[<a href="./papers/1/English Overview.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein function annotation using protein domain family resources.
			<b> Methods</b>, 2016.
			[<a href="./papers/1/Review_Domain_2016.pdf">PDF</a>]
		</font>
	</p>
</li>

</ol>

<font color="blue">CAFA Chanllenge (Evaluation Metrics)</font>

<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CAFA3.
			<b> Genome Biology</b>, 2019.
			[<a href="./papers/1/CAFA3_Genome Biology_2019.pdf">PDF</a>]
			[<a href="https://figshare.com/articles/Supplementary_data/8135393/3">dataset link</a>]
			[<a href="./papers/1/CAFA3_dataset.zip">dataset</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CAFA2.
			<b> Genome Biology</b>, 2016.
			[<a href="./papers/1/CAFA2_Genome Biology_2016.pdf">PDF</a>]
			[<a href="https://dx.doi.org/10.6084/m9.figshare.2059944.v1">dataset link</a>]
			[<a href="./papers/1/CAFA2_dataset.zip">dataset</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CAFA1.
			<b> Nature Methods</b>, 2013.
			[<a href="./papers/1/CAFA1_Nature_Methods_2013.pdf">PDF</a>]
			[<a href="https://biofunctionprediction.org/cafa/">office page</a>]
		</font>
	</p>
</li>

</ol>
<font color="blue">Toolbars</font>
<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			InterPro in 2022.
			<b> Nucleic Acids Research</b>, 2022.
			[<a href="./papers/1/InterPro_NAR_2022.pdf">PDF</a>]
			[<a href="https://www.ebi.ac.uk/interpro/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

</ol>

<font color="blue">Public Database</font>

<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein sequence database:
			<a href="https://www.uniprot.org">UniProt</a>.
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein structure database:
			<a href="https://www.rcsb.org">PDB</a>,
			<a href="https://alphafold.ebi.ac.uk/">AlphaFold database</a>.
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein function database:
			<a href="https://geneontology.org">Gene Ontology</a>,
			<a href="https://www.ebi.ac.uk/GOA/index">GOA</a>.
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein-ligand strcture database:
			<a href="https://zhanggroup.org/BioLiP/">BioLip</a>.
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Gene co-expression database:
			<a href="https://coxpresdb.jp">COXPRESdb</a>,
			<a href="https://atted.jp">ATTED-II</a>.
		</font>
	</p>
</li>

</ol>

<font color="blue">Template-Based Methods</font>
<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			QAUST: Protein function prediction using structure similarity, protein interaction, and functional motifs.
			<b> Genomics Proteomics Bioinformatics</b>, 2021.
			<b>Source: structure, protein-protein network, and functional motifs.</b>
			[<a href="./papers/1/QAUST_GPB_2021.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MLC: Metric learning on expression data for gene function prediction.
			<b> Bioinformatics</b>, 2020.
			<b>Source: gene expression.</b>
			[<a href="./papers/1/MLC_Bioinformatics_2020.pdf">PDF</a>]
			[<a href="https://github.com/stamakro/MLC" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			INGA 2.0: Improving protein function prediction for the dark proteome.
			<b> Nucleic Acids Research</b>, 2019.
			<b>Source: sequenc,protein-protein network, domain.</b>
			[<a href="./papers/1/INGA2.0_NAR-2019.pdf">PDF</a>]
			[<a href="https://inga.bio.unipd.it/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MetaGO: Predicting gene ontology of non-homologous proteins through low-resolution protein structure prediction and protein-protein network mapping.
			<b> Journal of Molecular Biology</b>, 2018.
			<b>Source: sequence, structure, and protein-protein network.</b>
			[<a href="./papers/1/MetaGO_Journal of Molecular Biology_2018.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/MetaGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			FunFams: Functional classification of CATH superfamilies: a domain-based approach for protein function annotation.
			<b> Bioinformatics</b>, 2017.
			<b>Source: family and domain.</b>
			[<a href="./papers/1/FunFams_Bioinformatics_2015.pdf">PDF</a>]
			[<a href="http://www.cathdb.info/search/by_funfhmmer/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			COFACTOR 2.0: Improved protein function prediction by combining structure, sequence and protein-protein interaction information.
			<b> Nucleic Acids Research</b>, 2017.
			<b>Source: sequence, structure, and protein-protein network.</b>
			[<a href="./papers/1/COFACTOR_Nucleic Acids Research_2017.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/COFACTOR/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GoFDR: A sequence alignment based method for predicting protein functions.
			<b> Methods</b>, 2017.
			<b>Source: sequence.</b>
			[<a href="./papers/1/GoFDR_Methods_2016.pdf">PDF</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			INGA: Protein function prediction combining interaction networks, domain assignments and sequence similarity.
			<b> Nucleic Acids Research</b>, 2015.
			<b>Source: sequenc,protein-protein network, domain.</b>
			[<a href="./papers/1/INGA_NAR_2015.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MS-kNN: Protein function prediction by integrating multiple data sources.
			<b> BMC Bioinformatics</b>, 2013.
			<b>Source: sequence, protein-protein network, and gene expression.</b>
			[<a href="./papers/1/MS-kNN_BMC Bioinformatics_2013.pdf">PDF</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			dcGO: A domain-centric solution to functional genomics via dcGO predictor.
			<b> BMC Bioinformatics</b>, 2013.
			<b>Source: domain.</b>
			[<a href="./papers/1/dcGO_BMC Bioinformatics_2013.pdf">PDF</a>]
			[<a href="https://supfam.org/SUPERFAMILY/dcGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			COFACTOR 1.0: An accurate comparative algorithm for structure-based protein function annotation.
			<b> Nucleic Acids Research</b>, 2012.
			<b>Source: structure.</b>
			[<a href="./papers/1/COFACTOR_Nucleic Acids Research_2012.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			FINDSITE: A combined evolution/structure-based approach to protein function prediction.
			<b> Briefings in Bioinformatics</b>, 2009.
			<b>Source: structure.</b>
			[<a href="./papers/1/FINDSITE_BIB_2009.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MultiPfam2GO: Predicting protein function from domain content.
			<b> Bioinformatics</b>, 2009.
			<b>Source: domain.</b>
			[<a href="./papers/1/MultiPfam2GO_Bioinformatics_2008.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Exploiting indirect neighbours and topological weight to predict protein function from protein–protein interactions.
			<b> Bioinformatics</b>, 2006.
			<b>Source: protein-protein network.</b>
			[<a href="./papers/1/PPI_Bioinformatics_2006.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ProFunc: A server for predicting protein function from 3D structure.
			<b> Nucleic Acids Research</b>, 2005.
			<b>Source: structure.</b>
			[<a href="./papers/1/ProFunc_NAR_2005.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Blast2GO: A universal tool for annotation, visualization and analysis in functional genomics research.
			<b> Bioinformatics</b>, 2005.
			<b>Source: sequence.</b>
			[<a href="./papers/1/Blast2GO_Bioinformatics_2005.pdf">PDF</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GOtcha: A new method for prediction of protein function assessed by the annotation of seven genomes.
			<b> BMC Bioinformatics</b>, 2004.
			<b>Source: sequence.</b>
			[<a href="./papers/1/GOtcha_BMC Bioinformatics_2004.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Global protein function prediction from protein-protein interaction networks.
			<b> Nature Biotechnology</b>, 2003.
			<b>Source: protein-protein network.</b>
			[<a href="./papers/1/PPI_Nature Biotechnology_2003.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Predicting protein function from protein/protein interaction data: a probabilistic approach.
			<b> Bioinformatics</b>, 2003.
			<b>Source: protein-protein network.</b>
			[<a href="./papers/1/PPI_Bionformatics_2003.pdf">PDF</a>]
		</font>
	</p>
</li>



</ol>

<font color="blue">Machine Learning-Based Methods</font>
<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NetGO: Improving large-scale protein function prediction with massive network information.
			<b> Nucleic Acids Research</b>, 2019.
			[<a href="./papers/1/NetGO_Nucleic Acids Research_2019.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GOLabeler: Improving sequence-based large-scale protein function prediction by learning to rank.
			<b> Bioinformatics</b>, 2018.
			[<a href="./papers/1/GOlabeler_Bioinformatics_2018.pdf">PDF</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			FFPred: an integrated feature-based function prediction server for vertebrate proteomes.
			<b> Nucleic Acids Research</b>, 2008.
			[<a href="./papers/1/FFPred_NAR_2008.pdf">PDF</a>]
		</font>
	</p>
</li>

</ol>

<font color="blue">Deep Learning-Based Methods</font>
<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PBiLSTM-FCN: Grain protein function prediction based on improved FCN and bidirectional LSTM.
			<b> Food Chemistry</b>, 2025.
			[<a href="./papers/1/PBiLSTM-FCN.pdf">PDF</a>]
			[<a href="https://github.com/Kun-Li-lab/PBiLSTM-FCN" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			SEGT‑GO: A graph transformer method based on PPI serialization and explanatory artificial intelligence for protein function prediction.
			<b> BMC Bioinformatics</b>, 2025.
			[<a href="./papers/1/SEGT‑GO.pdf">PDF</a>]
			[<a href="https://github.com/SpaceZore/SEGT-GO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MMSNet: Grain Protein Function Prediction Based on CNN and Residual Attention Mechanism with AlphaFold2 Structure Data.
			<b>  Applied Sciences</b>, 2025.
			[<a href="./papers/1/MMSNet.pdf">PDF</a>]
			[<a href="https://www.mdpi.com/2076-3417/15/4/1890" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepSS2GO: Protein function prediction from secondary structure.
			<b> Briefings in Bioinformatics</b>, 2024.
			[<a href="./papers/1/DeepSS2GO_BIB_2024.pdf">PDF</a>]
			[<a href="https://github.com/orca233/DeepSS2GO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MSF-PFP: A novel multisource feature fusion model for protein function prediction.
			<b> Journal of Chemical Information and Modeling</b>, 2024.
			[<a href="./papers/1/MSF-PFP_JCIM_2024.pdf">PDF</a>]
			[<a href="https://swanhub.co/TianGua/MSF-PFP" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Chemical-SA-BiLSTM: Grain protein function prediction based on self-attention mechanism and bidirectional LSTM.
			<b> Briefings in Bioinformatics</b>, 2023.
			[<a href="./papers/1/Chemical-SA-BiLSTM_BIB_2023.pdf">PDF</a>]
			[<a href="https://github.com/HwaTong/Chemical-SA-BiLSTM" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ProteInfer: Deep neural networks for protein functional inference.
			<b> eLife</b>, 2023.
			[<a href="./papers/1/ProteInfer_eLife_2023.pdf">PDF</a>]
			[<a href="https://github.com/google-research/proteinfer" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepPFP-CO: A deep learning framework for predicting protein functions with co-occurrence of GO terms.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2023.
			[<a href="./papers/1/DeepPFP-CO_TCBB_2023.pdf">PDF</a>]
			[<a href="https://csuligroup.com/DeepPFP/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TripletGO: Integrating transcript expression profiles with protein homology inferences for gene function prediction.
			<b> Genomics Proteomics Bioinformatics</b>, 2022.
			[<a href="./papers/1/TripletGO_Genomics Proteomics Bioinformatics_2022.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/TripletGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGOZero: Improving protein function prediction from sequence and zero-shot learning based on ontology axioms.
			<b> Bioinformatics</b>, 2022.
			[<a href="./papers/1/DeepGOZero_Bioiformatics_2022.pdf">PDF</a>]
			[<a href="https://github.com/bio-ontology-research-group/deepgozero" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NetGO 2.0: Improving large-scale protein function prediction with massive sequence, text, domain, family and network information.
			<b> Nucleic Acids Research</b>, 2021.
			[<a href="./papers/1/NetGO 2.0_Nucleic Acids Research_2021.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGOA: A deep learning framework for gene ontology annotations with sequence- and network-based information.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2021.
			[<a href="./papers/1/DeepGOA_TCBB_2021.pdf">PDF</a>]
			[<a href="https://github.com/CSUBioGroup/DeepGOA" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NetQuilt: Deep multispecies network-based protein function prediction using homology-informed network similarity.
			<b> Bioinformatics </b>, 2021.
			[<a href="./papers/1/NetQuilt_Bioinformatics_2021.pdf">PDF</a>]
			[<a href="https://github.com/nowittynamesleft/NetQuilt" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MultiPredGO: Deep multi-modal protein function prediction by amalgamating protein structure, sequence, and interaction information.
			<b> IEEE Journal of Biomedical and Health Informatics </b>, 2021.
			[<a href="./papers/1/MultiPredGO_JBHI_2021.pdf">PDF</a>]
			[<a href="https://github.com/SwagarikaGiri/Multi-PredGO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TALE: Transformer-based protein function Annotation with joint sequence-Label Embedding.
			<b> Bioinformatics</b>, 2021.
			[<a href="./papers/1/TALE_Bioinformatics_2021.pdf">PDF</a>]
			[<a href="https://github.com/Shen-Lab/TALE" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGraphGO: graph neural network for large-scale, multispecies protein function prediction.
			<b> Bioinformatics</b>, 2021.
			[<a href="./papers/1/DeepGraphGO_Bioinformatics_2021.pdf">PDF</a>]
			[<a href="https://github.com/yourh/DeepGraphGO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Deep_CNN_LSTM_GO: Protein function prediction from amino-acid sequences.
			<b> Computational Biology and Chemistry</b>, 2021.
			[<a href="./papers/1/Deep_CNN_LSTM_GO_Computational Biology and Chemistry_2021.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			FFPred-GAN: Protein function prediction is improved by creating synthetic feature samples with generative adversarial networks.
			<b> Nature Machine Intelligence</b>, 2020.
			[<a href="./papers/1/FFPred-GAN_Nature Machine Intelligence_2020.pdf">PDF</a>]
			[<a href="https://github.com/psipred/FFPredGAN" target=_blank>Code</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepAdd: Protein function prediction from k-mer embedding and additional features.
			<b> Computational Biology and Chemistry</b>, 2020.
			[<a href="./papers/1/DeepAdd_CBC_2020.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGOPlus: Improved protein function prediction from sequence.
			<b> Bioinformatics</b>, 2020.
			[<a href="./papers/1/DeepGOPlus_Bioinformatics_2020.pdf">PDF</a>]
			[<a href="https://deepgo.cbrc.kaust.edu.sa/deepgo/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGO: Predicting protein functions from sequence and interactions using a deep ontology-aware classifier.
			<b> Bioinformatics</b>, 2018.
			[<a href="./papers/1/DeepGO_Bioinformatics_2018.pdf">PDF</a>]
			[<a href="https://github.com/bio-ontology-research-group/deepgo" target=_blank>Code</a>]
		</font>
	</p>
</li>
</ol>

<font color="blue">Pre-Trained Model-Based Methods</font>
<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			AnnoPRO: A strategy for protein function annotation based on multi-scale protein representation and a hybrid deep learning of dual-path encoding.
			<b> Genome Biology</b>, 2024.
			[<a href="./papers/1/AnnoPRO_Genome Biology_2024.pdf">PDF</a>]
			[<a href="https://github.com/idrblab/AnnoPRO" target=_blank>Code</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DualNetGO: A dual network model for protein function prediction via effective feature selection.
			<b> Bioinformatics</b>, 2024.
			[<a href="./papers/1/DualNetGO_Bioinformatics_2024.pdf">PDF</a>]
			[<a href=" https://github.com/georgedashen/DualNetGO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Domain-PFP: Protein function prediction using function-aware domain embedding representations.
			<b> Communications Biology</b>, 2023.
			[<a href="./papers/1/Domain-PFP_Communications Biology_2023.pdf">PDF</a>]
			[<a href="https://github.com/kiharalab/Domain-PFP" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CFAGO: Cross-fusion of network and attributes based on attention mechanism for protein function prediction.
			<b> Bioinformatics</b>, 2023.
			[<a href="./papers/1/CFAGO_Bioinformatics_2023.pdf">PDF</a>]
			[<a href="http://bliulab.net/CFAGO/" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MELISSA: Semi-supervised embedding for protein function prediction across multiple networks
			<b> bioRxiv</b>, 2023.
			[<a href="./papers/1/MELISSA_bioRxiv_2023.pdf">PDF</a>]
			[<a href="https://github.com/XiaozheHu/melissa" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			HiFun: Homology independent protein function prediction by a novel protein-language self-attention model.
			<b> Briefings in Bioinformatics</b>, 2023.
			[<a href="./papers/1/HiFun_BIB_2023.pdf">PDF</a>]
			[<a href="http://www.unimd.org/HiFun" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PredGO: Large-scale predicting protein functions through heterogeneous feature fusion.
			<b> Briefings in Bioinformatics</b>, 2023.
			[<a href="./papers/1/PredGO_BIB_2023.pdf">PDF</a>]
			[<a href="http://predgo.denglab.org/" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MGEGFP: A multi-view graph embedding method for gene function prediction based on adaptive estimation with GCN.
			<b> Briefings in Bioinformatics</b>, 2023.
			[<a href="./papers/1/MGEGFP_BIB_2023.pdf">PDF</a>]
			[<a href="https://github.com/zhanglabNKU/MGEGFP" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MMSMAPlus: A multi-view multi-scale multi-attention embedding model for protein function prediction.
			<b> Briefings in Bioinformatics</b>, 2023.
			[<a href="./papers/1/MMSMAPlus_BIB_2023.pdf">PDF</a>]
			[<a href="https://github.com/wzy-2020/MMSMAPlus" target=_blank>Code</a>]
		</font>
	</p>
</li>



<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PFmulDL: A novel strategy enabling multi-class and multi-label protein function annotation by integrating diverse deep learning methods.
			<b> Computers in Biology and Medicine</b>, 2022.
			[<a href="./papers/1/PFmulDL_Computers in Biology and Medicine_2022.pdf">PDF</a>]
			[<a href="https://github.com/idrblab/PFmulDL" target=_blank>Code</a>]

		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepFRI: Structure-based protein function prediction using graph convolutional networks.
			<b> Nature Communications</b>, 2021.
			[<a href="./papers/1/DeepFRI_Nature Communications _2021.pdf">PDF</a>]
			[<a href="https://beta.deepfri.flatironinstitute.org/" target=_blank>Web Server</a>]
			[<a href="https://github.com/flatironinstitute/DeepFRI/" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			deepNF: Deep network fusion for protein function prediction.
			<b> Bioinformatics</b>, 2018.
			[<a href="./papers/1/deepNF_Bioinfortics_2018.pdf">PDF</a>]
			[<a href="https://github.com/VGligorijevic/deepNF" target=_blank>Code</a>]
		</font>
	</p>
</li>

</ol>

<font color="blue">Large Language Model-Based Methods</font>
<ol>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DPFunc: Accurately predicting protein function via deep learning with domainguided structure information.
			<b> Nature Communications</b>, 2025.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/DPFunc.pdf">PDF</a>]
			[<a href="https://github.com/CSUBioGroup/DPFunc" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GOPhage: Protein function annotation for bacteriophages by integrating the genomic context.
			<b> Briefings in Bioinformatics</b>, 2025.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/GOPhage.pdf">PDF</a>]
			[<a href="https://github.com/jiaojiaoguan/GOPhage" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepFMB: Enhancing Protein Function Prediction Through the Fusion of Multi-Type Biological Knowledge With Protein Language Model and Graph Neural Network.
			<b> IEEE Transactions on Computational Biology and Bioinformatics</b>, 2025.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/DeepFMB.pdf">PDF</a>]
			[<a href="https://github.com/CSUBioGroup/DeepFMB" target=_blank>Code</a>]
		</font>
	</p>
</li>



<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepMFFGO: A Protein Function Prediction Method for Large-Scale Multifeature Fusion.
			<b> Journal of Chemical Information and Modeling</b>, 2025.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/DeepMFFGO.pdf">PDF</a>]
			[<a href="https://github.com/AI-HPC-Research-Team/ProtFAD" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepMVD: A Novel Multiview Dynamic Feature Fusion Model for Accurate Protein Function Prediction.
			<b> Journal of Chemical Information and Modeling</b>, 2025.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/DeepMVD.pdf">PDF</a>]
			[<a href="http://swanhub.co/scl/DeepMVD" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ProtFAD: Aligning sequence and structure representations leveraging protein domains for function prediction.
			<b> Expert Systems With Applications</b>, 2025.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/ProtFAD.pdf">PDF</a>]
			[<a href="https://github.com/AI-HPC-Research-Team/ProtFAD" target=_blank>Code</a>]
		</font>
	</p>
</li>

	<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MMPFP: A multimodal model for protein function prediction.
			<b> Scientific Reports</b>, 2025.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/MMPFP.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGO-SE: Protein function prediction as approximate semantic entailment.
			<b> Nature Machine Intelligence</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/DeepGO-SE_Nature Machine Intelligence_2024.pdf">PDF</a>]
			[<a href="https://github.com/bio-ontology-research-group/deepgo2" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PhiGnet: Accurate prediction of protein function using statistics-informed graph networks.
			<b> Nature Communications</b>, 2024.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/PhiGnet_NC_2024.pdf">PDF</a>]
			[<a href="https://zenodo.org/records/12496869" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			InterLabelGO+: Unraveling label correlations in protein function prediction.
			<b> Bioinformatics</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/InterLabelGO_BIoinformatics_2024.pdf">PDF</a>]
			[<a href="https://github.com/QuanEvans/InterLabelGO" target=_blank>Code</a>]
			[<a href="https://seq2fun.dcmb.med.umich.edu/InterLabelGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GORetriever: Reranking protein-description-based GO candidates by literature-driven deep information retrieval for protein function annotation.
			<b> Bioinformatics</b>, 2024.
			<b> Model: PubMedBERT.</b>
			[<a href="./papers/1/GORetriever.pdf">PDF</a>]
			[<a href="https://github.com/ZhuLab-Fudan/GORetriever" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TAWFN: A deep learning framework for protein function prediction.
			<b> Bioinformatics</b>, 2024.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/TAWFN.pdf">PDF</a>]
			[<a href="https://github.com/ss0830/TAWFN" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepMTC: Deep learning model for protein multi-label subcellular localization and function prediction based on multi-task collaborative training.
			<b> Briefings in Bioinformatics</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/DeepMTC.pdf">PDF</a>]
			[<a href="https://github.com/ghli16/DeepMTC" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PU-GO: Predicting protein functions using positive-unlabeled ranking with ontology-based priors.
			<b> Bioinformatics</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/PU-GO.pdf">PDF</a>]
			[<a href="https://github.com/bio-ontology-research-group/PU-GO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GPSFun: Geometry-aware protein sequence function predictions with language models.
			<b> Nucleic Acids Research</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/GPSFun_NAR_2024.pdf">PDF</a>]
			[<a href="https://bio-web1.nscc-gz.cn/app/GPSFun" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			SUPERMAGO: Protein Function Prediction Based on Transformer Embeddings.
			<b> Proteins: Structure, Function, and Bioinformatics</b>, 2024.
			<b> Model: ESM2 and ProtTrans.</b>
			[<a href="./papers/1/SUPERMAGO.pdf">PDF</a>]
			[<a href="https://github.com/gabrielbianchin/SUPERMAGO" target=_blank>Code</a>],
			[<a href="https://supermago.ic.unicamp.br" target=_blank>Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GO-LTR: Protein function prediction through multi‑view multi‑label latent tensor reconstruction.
			<b> BMC Bioinformatics</b>, 2024.
			<b> Model: ProtTrans.</b>
			[<a href="./papers/1/GO-LTR_BMC_2024.pdf">PDF</a>]
			[<a href="https://github.com/aalto-ics-kepaco/GO-LTR-prediction" target=_blank>Code</a>]
		</font>
	</p>
</li>



	<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TransFew: Improving protein function prediction by learning and integrating representations of protein sequences and function labels.
			<b> Bioinformatics Advances</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/TransFew_BA_2024.pdf">PDF</a>]
			[<a href="https://github.com/BioinfoMachineLearning/TransFew" target=_blank>Code</a>]
		</font>
	</p>

	<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			StarFunc+: Fusing template-based and deep learning approaches for accurate protein function prediction.
			<b> bioRxiv</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/StarFunc_BioRxiv_2024.pdf">PDF</a>]
			[<a href="https://seq2fun.dcmb.med.umich.edu/StarFunc/" target=_blank>Web Server</a>]
		</font>
	</p>
   </li>



</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGOMeta: Predicting functions for microbes.
			<b> bioRxiv</b>, 2024.
			<b> Model: ESM2.</b>
			[<a href="./papers/1/DeepGOMeta_BioRxiv_2024.pdf">PDF</a>]
			[<a href="https://github.com/bio-ontology-research-group/deepgometa" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GNNGO3D: Protein function prediction based on 3D structure and functional hierarchy learning.
			<b> IEEE Transactions on Knowledge and Data Engineering</b>, 2023.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/GNNGO3D_TKDE_2023.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Struct2GO: Protein function prediction based on graph pooling algorithm and AlphaFold2 structure information.
			<b> Bioinformatics</b>, 2023.
			<b> Model: SeqVec.</b>
			[<a href="./papers/1/Struct2GO_Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/lyjps/Struct2GO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			SPROF-GO: Fast and accurate protein function prediction from sequence through pretrained language model and homology-based label diffusion.
			<b> Briefings in Bioinformatics</b>, 2023.
			<b> Model: ProtTrans.</b>
			[<a href="./papers/1/SPROF-GO_Briefings in Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/biomed-AI/SPROF-GO/" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			HNetGO: Protein function prediction via heterogeneous network transformer.
			<b> Briefings in Bioinformatics</b>, 2023.
			<b> Model: SeqVec.</b>
			[<a href="./papers/1/HNetGO_BIB_2023.pdf">PDF</a>]
			[<a href="https://github.com/BIOGOHITSZ/HNetGO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NetGO 3.0: A protein language model improves large-scale functional Annotations.
			<b> Genomics Proteomics Bioinformatics</b>, 2023.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/NetGO 3.0_Genomics Proteomics Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://dmiip.sjtu.edu.cn/ng3.0" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			HEAL: Hierarchical graph transformer with contrastive learning for protein function prediction.
			<b> Bioinformatics</b>, 2023.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/HEAL_Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/ZhonghuiGu/HEAL" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TransFun: Combining protein sequences and structures with transformers and equivariant graph neural networks to predict protein function.
			<b> Bioinformatics</b>, 2023.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/TransFun_Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/jianlin-cheng/TransFun" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TEMPROT: Protein function annotation using transformers embeddings and homology search.
			<b> BMC Bioinformatics</b>, 2023.
			<b> Model: ProtBERT-BFD.</b>
			[<a href="./papers/1/TEMPROT_BMC Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/gabrielbianchin/TEMPROT/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PANDA2: Protein function prediction using graph neural networks.
			<b> NAR Genomics and Bioinformatics</b>, 2022.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/PANDA2_NAR Genomics and Bioinformatics_2022.pdf">PDF</a>]
			[<a href="http://dna.cs.miami.edu/PANDA2/" target=_blank>Web Server</a>]

		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ATGO: Integrating unsupervised language model with triplet neural networks for protein gene ontology prediction.
			<b> PLoS Computational Biology</b>, 2022.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/ATGO_PLoS Computational Biology_2022.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/ATGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GAT-GO: Accurate protein function prediction via graph attention networks with predicted structure information.
			<b> Briefings in Bioinformatics</b>, 2022.
			<b> Model: ESM-1b.</b>
			[<a href="./papers/1/GATGO_Briefings in Bioinformatics_2022.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GOPredSim: Embeddings from deep learning transfer GO annotations beyond homology.
			<b> Scientific Reports</b>, 2021.
			<b> Model: SeqVec.</b>
			[<a href="./papers/1/GOPredSim_Scientific Reports_2021.pdf">PDF</a>]
			[<a href="https://embed.protein.properties" target=_blank>Web Server</a>]
			[<a href="https://github.com/Rostlab/goPredSim" target=_blank>Code</a>]
		</font>
	</p>
</li>
</ol>

		</td>
		</tr>
	</table>
</div>
