<style>
<!--
 li.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	font-size:10.5pt;
	font-family:"Times New Roman";
	margin-left:0cm; margin-right:0cm; margin-top:0cm}
-->
</style>

<div>
	<table border="0" id="table1" bgcolor="#FFFFFF">
<tr>
	<td valign="baseline" colspan="3" bgcolor="#ADD8E6">
		<p><b>
			<font face="Times" size="3">
				<a name="2023">Protein Function Prediction</a>
			</font>
		</b></p>
	</td>
</tr>
		<tr>
		<td valign="baseline" colspan="3">
		<ul>



<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CFAGO: Cross-fusion of network and attributes based on attention mechanism for protein function prediction.
			<b> Bioinformatics</b>, 2023.
			[<a href="./papers/1/CFAGO_Bioinformatics_2023.pdf">PDF</a>]
			[<a href="http://bliulab.net/CFAGO/" target=_blank>Code</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			SPROF-GO: Fast and accurate protein function prediction from sequence through pretrained language model and homology-based label diffusion.
			<b> Briefings in Bioinformatics</b>, 2023.
			[<a href="./papers/1/SPROF-GO_Briefings in Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/biomed-AI/SPROF-GO/" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NetGO 3.0: A protein language model improves large-scale functional Annotations.
			<b> Genomics Proteomics Bioinformatics</b>, 2023.
			[<a href="./papers/1/NetGO 3.0_Genomics Proteomics Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://dmiip.sjtu.edu.cn/ng3.0" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			HEAL: Hierarchical graph transformer with contrastive learning for protein function prediction.
			<b> Bioinformatics</b>, 2023.
			[<a href="./papers/1/HEAL_Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/ZhonghuiGu/HEAL" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepPFP-CO: A deep learning framework for predicting protein functions with co-occurrence of GO terms.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2023.
			[<a href="./papers/1/DeepPFP-CO_TCBB_2023.pdf">PDF</a>]
			[<a href="https://csuligroup.com/DeepPFP/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TEMPROT: Protein function annotation using transformers embeddings and homology search.
			<b> BMC Bioinformatics</b>, 2023.
			[<a href="./papers/1/TEMPROT_BMC Bioinformatics_2023.pdf">PDF</a>]
			[<a href="https://github.com/gabrielbianchin/TEMPROT/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>



<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TripletGO: Integrating transcript expression profiles with protein homology inferences for gene function prediction.
			<b> Genomics Proteomics Bioinformatics</b>, 2022.
			[<a href="./papers/1/TripletGO_Genomics Proteomics Bioinformatics_2022.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/TripletGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			PANDA2: Protein function prediction using graph neural networks.
			<b> NAR Genomics and Bioinformatics</b>, 2022.
			[<a href="./papers/1/PANDA2_NAR Genomics and Bioinformatics_2022.pdf">PDF</a>]
			[<a href="http://dna.cs.miami.edu/PANDA2/" target=_blank>Web Server</a>]

		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GAT-GO: Accurate protein function prediction via graph attention networks with predicted structure information.
			<b> Briefings in Bioinformatics</b>, 2022.
			[<a href="./papers/1/GATGO_Briefings in Bioinformatics_2022.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ATGO: Integrating unsupervised language model with triplet neural networks for protein gene ontology prediction.
			<b> PLoS Computational Biology</b>, 2022.
			[<a href="./papers/1/ATGO_PLoS Computational Biology_2022.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/ATGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NetGO 2.0: Improving large-scale protein function prediction with massive sequence, text, domain, family and network information.
			<b> Nucleic Acids Research</b>, 2021.
			[<a href="./papers/1/NetGO 2.0_Nucleic Acids Research_2021.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepFRI: Structure-based protein function prediction using graph convolutional networks.
			<b> Nature Communications</b>, 2021.
			[<a href="./papers/1/DeepFRI_Nature Communications _2021.pdf">PDF</a>]
			[<a href="https://beta.deepfri.flatironinstitute.org/" target=_blank>Web Server</a>]
			[<a href="https://github.com/flatironinstitute/DeepFRI/" target=_blank>Code</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGOA: A deep learning framework for gene ontology annotations with sequence- and network-based information.
			<b> IEEE/ACM Transactions on Computational Biology and Bioinformatics</b>, 2021.
			[<a href="./papers/1/DeepGOA_TCBB_2021.pdf">PDF</a>]
			[<a href="https://github.com/CSUBioGroup/DeepGOA" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			TALE: Transformer-based protein function Annotation with joint sequence-Label Embedding.
			<b> Bioinformatics</b>, 2021.
			[<a href="./papers/1/TALE_Bioinformatics_2021.pdf">PDF</a>]
			[<a href="https://github.com/Shen-Lab/TALE" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GOPredSim: Embeddings from deep learning transfer GO annotations beyond homology.
			<b> Scientific Reports</b>, 2021.
			[<a href="./papers/1/GOPredSim_Scientific Reports_2021.pdf">PDF</a>]
			[<a href="https://embed.protein.properties" target=_blank>Web Server</a>]
			[<a href="https://github.com/Rostlab/goPredSim" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGraphGO: graph neural network for large-scale, multispecies protein function prediction.
			<b> Bioinformatics</b>, 2021.
			[<a href="./papers/1/DeepGraphGO_Bioinformatics_2021.pdf">PDF</a>]
			[<a href="https://github.com/yourh/DeepGraphGO" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			FFPred-GAN: Protein function prediction is improved by creating synthetic feature samples with generative adversarial networks.
			<b> Nature Machine Intelligence</b>, 2020.
			[<a href="./papers/1/FFPred-GAN_Nature Machine Intelligence_2020.pdf">PDF</a>]
			[<a href="https://github.com/psipred/FFPredGAN" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MLC: Metric learning on expression data for gene function prediction.
			<b> Bioinformatics</b>, 2020.
			[<a href="./papers/1/MLC_Bioinformatics_2020.pdf">PDF</a>]
			[<a href="https://github.com/stamakro/MLC" target=_blank>Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGOPlus: Improved protein function prediction from sequence.
			<b> Bioinformatics</b>, 2020.
			[<a href="./papers/1/DeepGOPlus_Bioinformatics_2020.pdf">PDF</a>]
			[<a href="https://deepgo.cbrc.kaust.edu.sa/deepgo/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Review: The CAFA challenge reports improved protein function prediction and new functional annotations for hundreds of genes through experimental screens.
			<b> Genome Biology</b>, 2019.
			[<a href="./papers/1/CAFA3_Genome Biology_2019.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			NetGO: Improving large-scale protein function prediction with massive network information.
			<b> Nucleic Acids Research</b>, 2019.
			[<a href="./papers/1/NetGO_Nucleic Acids Research_2019.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			MetaGO: Predicting gene ontology of non-homologous proteins through low-resolution protein structure prediction and protein-protein network mapping.
			<b> Journal of Molecular Biology</b>, 2018.
			[<a href="./papers/1/MetaGO_Journal of Molecular Biology_2018.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/MetaGO/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			GOLabeler: Improving sequence-based large-scale protein function prediction by learning to rank.
			<b> Bioinformatics</b>, 2018.
			[<a href="./papers/1/GOlabeler_Bioinformatics_2018.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DeepGO: Predicting protein functions from sequence and interactions using a deep ontology-aware classifier.
			<b> Bioinformatics</b>, 2018.
			[<a href="./papers/1/DeepGO_Bioinformatics_2018.pdf">PDF</a>]
			[<a href="https://github.com/bio-ontology-research-group/deepgo" target=_blank>Code</a>]
		</font>
	</p>
</li>


<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			COFACTOR 2.0: Improved protein function prediction by combining structure, sequence and protein-protein interaction information.
			<b> Nucleic Acids Research</b>, 2017.
			[<a href="./papers/1/COFACTOR_Nucleic Acids Research_2017.pdf">PDF</a>]
			[<a href="https://zhanggroup.org/COFACTOR/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			FunFams: Functional classification of CATH superfamilies: a domain-based approach for protein function annotation.
			<b> Bioinformatics</b>, 2017.
			[<a href="./papers/1/FunFams_Bioinformatics_2015.pdf">PDF</a>]
			[<a href="http://www.cathdb.info/search/by_funfhmmer/" target=_blank>Web Server</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Evaluation: A large-scale evaluation of computational protein function prediction.
			<b> Nature Methods</b>, 2013.
			[<a href="./papers/1/Evaluation_Nature_Methods_2013.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			COFACTOR 1.0: An accurate comparative algorithm for structure-based protein function annotation.
			<b> Nucleic Acids Research</b>, 2012.
			[<a href="./papers/1/COFACTOR_Nucleic Acids Research_2012.pdf">PDF</a>]
		</font>
	</p>
</li>



		</ul>
		</td>
		</tr>
	</table>
</div>
